ietf-eth-te-topology.yang

```diff
@@ -234,12 +288,15 @@ module ietf-eth-te-topology {
           description
             "Service classification capabilities based on the second
              VLAN tag, supported by the ETH LTP.";
-        /*
-         * Open issue: indicates that second-tag-classification
-         * can be True only if outer-tag-classification is also True.
-        */
           leaf second-tag-classification {
             type boolean;
+            must ". = 'false' or " 
+               + "../../vlan-tag-classification = 'true'" {
+              description
+                "VLAN service classification based on the second 
+                VLAN tag can be supported only when VLAN service 
+                classification";
+            }
```

Comment:
Need to check (testing some JSON examples) the correctness of this must statement

Reply:
See issue #17